sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ItemOperations.java

```diff
@@ -11,7 +11,7 @@
 /**
  * Note: although this class is public it is not part of our public API and may change in future.
  */
-public class ItemOperations {
+final public class ItemOperations {
```

Comment:
Is it a first class API (in public namespace)? Or will it be internal for now and later we will revisit on how to make it first class?

Reply:
we will move it to implementation package.

Reply:
moved to implementation.